config/config/sc-config.yaml

```diff
@@ -502,7 +502,7 @@ externalTrafficPolicy:
     prometheusWc: false
 
 nfsProvisioner:
-  server: ""
+  server: "set-me"
```

Comment:
I'm a bit unsure about this. This means that I, as a user, I will get an error even if I don't use the NFS storage provider, right? Maybe a validation inside the Helm Chart would be more appropriate?

Reply:
Yes, you'll see a warning.
That is one of the drawbacks with the current validation technque that we are employing, but if you want i can revert back to `""`. Nobody should use an nfs server anymore.

Reply:
Let's keep `""` for now and pick up the discussion on whether we want to finally move the storage providers out.